<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>rankboard</class>
 <widget class="QWidget" name="rankboard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>251</red>
        <green>248</green>
        <blue>239</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>251</red>
        <green>248</green>
        <blue>239</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>251</red>
        <green>248</green>
        <blue>239</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>251</red>
        <green>248</green>
        <blue>239</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>251</red>
        <green>248</green>
        <blue>239</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>251</red>
        <green>248</green>
        <blue>239</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>251</red>
        <green>248</green>
        <blue>239</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>251</red>
        <green>248</green>
        <blue>239</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>251</red>
        <green>248</green>
        <blue>239</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>2048</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(251, 248, 239);</string>
  </property>
  <widget class="QLabel" name="lblIcon">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>45</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/images/icon.jpeg);
border-radius: 5px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lblTitle">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>70</y>
     <width>150</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Algerian</family>
     <pointsize>36</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(115, 110, 100);</string>
   </property>
   <property name="text">
    <string>2048</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTableView" name="tbvRanklist">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>180</y>
     <width>700</width>
     <height>480</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(115, 110, 100);
background-color: rgb(251, 248, 239);
selection-color: rgb(115, 110, 100);
selection-background-color: rgb(219, 210, 200);</string>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::NoSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <property name="sortingEnabled">
    <bool>false</bool>
   </property>
   <attribute name="horizontalHeaderCascadingSectionResizes">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderStretchLastSection">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QLabel" name="lblRanklist">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>60</y>
     <width>300</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>隶书</family>
     <pointsize>48</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(115, 110, 100);</string>
   </property>
   <property name="text">
    <string>排行榜</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pBtnRet">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>75</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>隶书</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="toolTip">
    <string>返回上一级</string>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	border-image: url(:/interface/images/home.png);
	background-color: rgb(251, 248, 239);
	border-radius: 5px;
}

QPushButton:hover
{
	background-color: rgb(219, 210, 200);
}

QPushButton:pressed
{
	background-color: rgb(187, 173, 160);
	padding-left: 3px;
	padding-top: 3px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lblGoldImg">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>275</y>
     <width>340</width>
     <height>340</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>隶书</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/medal/images/gold.png);
background-color: rgba(251, 248, 239, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblBronzeImg">
   <property name="geometry">
    <rect>
     <x>365</x>
     <y>365</y>
     <width>160</width>
     <height>160</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>隶书</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/medal/images/bronze.png);
background-color: rgba(251, 248, 239, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblSilverImg">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>345</y>
     <width>200</width>
     <height>200</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>隶书</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/medal/images/silver.png);
background-color: rgba(251, 248, 239, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pBtn4">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>187</y>
     <width>60</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	color: rgb(115, 110, 100);
	background-color: rgb(251, 248, 239);
	border-radius: 5px;
}

QPushButton:hover
{
	background-color: rgb(219, 210, 200);
}

QPushButton:pressed
{
	background-color: rgb(187, 173, 160);
	padding-left: 3px;
	padding-top: 3px;
}</string>
   </property>
   <property name="text">
    <string>4×4</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pBtn5">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>187</y>
     <width>60</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	color: rgb(115, 110, 100);
	background-color: rgb(251, 248, 239);
	border-radius: 5px;
}

QPushButton:hover
{
	background-color: rgb(219, 210, 200);
}

QPushButton:pressed
{
	background-color: rgb(187, 173, 160);
	padding-left: 3px;
	padding-top: 3px;
}</string>
   </property>
   <property name="text">
    <string>5×5</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lblGoldName">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>585</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255,217,102);
background-color: rgba(251, 248, 239, 0);</string>
   </property>
   <property name="text">
    <string>gold name</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblGoldScore">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>615</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255,217,102);
background-color: rgba(251, 248, 239, 0);</string>
   </property>
   <property name="text">
    <string>gold score</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblSliverScore">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>560</y>
     <width>180</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(193,193,193);
background-color: rgba(251, 248, 239, 0);</string>
   </property>
   <property name="text">
    <string>silver score</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblSliverName">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>535</y>
     <width>180</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(193,193,193);
background-color: rgba(251, 248, 239, 0);</string>
   </property>
   <property name="text">
    <string>sliver name</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblBronzeScore">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>537</y>
     <width>170</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(199,145,68);
background-color: rgba(251, 248, 239, 0);</string>
   </property>
   <property name="text">
    <string>bronze score</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblBronzeName">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>517</y>
     <width>170</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(199,145,68);
background-color: rgba(251, 248, 239, 0);</string>
   </property>
   <property name="text">
    <string>bronze name</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblNo">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>180</y>
     <width>700</width>
     <height>480</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>隶书</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(115, 110, 100);
background-color: rgba(251, 248, 239, 0);</string>
   </property>
   <property name="text">
    <string>暂无任何排行榜数据</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pBtn3">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>187</y>
     <width>60</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	color: rgb(115, 110, 100);
	background-color: rgb(251, 248, 239);
	border-radius: 5px;
}

QPushButton:hover
{
	background-color: rgb(219, 210, 200);
}

QPushButton:pressed
{
	background-color: rgb(187, 173, 160);
	padding-left: 3px;
	padding-top: 3px;
}</string>
   </property>
   <property name="text">
    <string>3×3</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pBtn6">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>237</y>
     <width>60</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	color: rgb(115, 110, 100);
	background-color: rgb(251, 248, 239);
	border-radius: 5px;
}

QPushButton:hover
{
	background-color: rgb(219, 210, 200);
}

QPushButton:pressed
{
	background-color: rgb(187, 173, 160);
	padding-left: 3px;
	padding-top: 3px;
}</string>
   </property>
   <property name="text">
    <string>6×6</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pBtn7">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>237</y>
     <width>60</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	color: rgb(115, 110, 100);
	background-color: rgb(251, 248, 239);
	border-radius: 5px;
}

QPushButton:hover
{
	background-color: rgb(219, 210, 200);
}

QPushButton:pressed
{
	background-color: rgb(187, 173, 160);
	padding-left: 3px;
	padding-top: 3px;
}</string>
   </property>
   <property name="text">
    <string>7×7</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pBtn8">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>237</y>
     <width>60</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	color: rgb(115, 110, 100);
	background-color: rgb(251, 248, 239);
	border-radius: 5px;
}

QPushButton:hover
{
	background-color: rgb(219, 210, 200);
}

QPushButton:pressed
{
	background-color: rgb(187, 173, 160);
	padding-left: 3px;
	padding-top: 3px;
}</string>
   </property>
   <property name="text">
    <string>8×8</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>lblIcon</zorder>
  <zorder>lblTitle</zorder>
  <zorder>tbvRanklist</zorder>
  <zorder>lblRanklist</zorder>
  <zorder>pBtnRet</zorder>
  <zorder>lblGoldImg</zorder>
  <zorder>lblBronzeImg</zorder>
  <zorder>pBtn4</zorder>
  <zorder>pBtn5</zorder>
  <zorder>lblSilverImg</zorder>
  <zorder>lblGoldName</zorder>
  <zorder>lblGoldScore</zorder>
  <zorder>lblSliverScore</zorder>
  <zorder>lblSliverName</zorder>
  <zorder>lblBronzeScore</zorder>
  <zorder>lblBronzeName</zorder>
  <zorder>lblNo</zorder>
  <zorder>pBtn3</zorder>
  <zorder>pBtn6</zorder>
  <zorder>pBtn7</zorder>
  <zorder>pBtn8</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
